browser/ui/sidebar/sidebar_controller.cc

```diff
@@ -129,6 +131,28 @@ void SidebarController::ActivatePanelItem(
   panel_ui->Show(sidebar::SidePanelIdFromSideBarItemType(panel_item));
 }
 
+void SidebarController::ActivatePanelItem(const SidebarItem& item) {
+  CHECK(item.OpenInPanel());
+
+  if (!base::FeatureList::IsEnabled(sidebar::features::kSidebarMobileView)) {
+    CHECK_NE(SidebarItem::BuiltInItemType::kNone, item.built_in_item_type);
+    ActivatePanelItem(item.built_in_item_type);
+    return;
+  }
+
+  if (IsMobileViewItem(item)) {
+    auto* panel_ui = SidePanelUI::GetSidePanelUIForBrowser(browser_);
+    if (panel_ui) {
```

Comment:
super nit: can be inlined

```suggestion
    if (auto* panel_ui = SidePanelUI::GetSidePanelUIForBrowser(browser_)) {
```